web-app/src/app/services/feeds/index.ts

```diff
@@ -0,0 +1,201 @@
+import createClient, { type Middleware } from 'openapi-fetch';
+import type { paths } from './types';
+import { type AllFeedType } from './utils';
+
+const API_BASE_URL = 'https://api-dev.mobilitydatabase.org';
```

Comment:
Here, we can use a REACT PARAMETER something like `REACT_APP_API_BASE_URL`. It can be consumed with:
```
getEnvConfig('REACT_APP_API_BASE_URL');
```
_This can be done in a different PR as we are not consuming the API yet_